SM/internal/transport/handler/userList.go

```diff
@@ -61,8 +68,26 @@ func GetUserListByRole(p handler_utils.Params) gin.HandlerFunc {
 			logger.RequestLogger(p.Log, reqParams, handlerName, "Error", err)
 			return
 		}
+		usersOut, err := usersListCovert(users)
+		if err != nil {
+			logger.RequestLogger(p.Log, reqParams, handlerName, "Error", err)
+			return
+		}
+		logger.RequestLogger(p.Log, reqParams, handlerName, "Successfully", nil)
 		c.JSON(http.StatusOK, gin.H{
-			userRoleParam: users,
+			userRoleParam: usersOut,
 		})
 	}
 }
+
+func usersListCovert(users []postgres.User) ([]interface{}, error) {
+	var usersOut []interface{}
+	for i := range users {
+		userOut, err := handler_output.ConvertToOut(users[i])
+		usersOut = append(usersOut, userOut)
+		if err != nil {
+			return nil, err
```

Comment:
you can return empty struct here, it allow you to get rid of interface{} in signature